include/wx/qt/glcanvas.h

```diff
@@ -87,4 +87,7 @@ class WXDLLIMPEXP_GL wxGLCanvas : public wxGLCanvasBase
     wxDECLARE_CLASS(wxGLCanvas);
 };
 
+#include <QtWidgets/QGestureRecognizer>
```

Comment:
@bdbcat if these include statements don't need to be exposed in the public header they should be moved into the .cpp file that needs them.